demos/speech_recognition_wav2vec_demo/python/speech_recognition_wav2vec_demo.py

```diff
@@ -51,23 +53,25 @@ class Wav2Vec:
     words_delimiter = '|'
     pad_token = '<pad>'
 
-    def __init__(self, core, model_path, input_shape, device, vocab_file):
+    def __init__(self, core, model_path, input_shape, device, vocab_file, dynamic_flag):
         log.info('Reading model {}'.format(model_path))
         model = core.read_model(model_path)
         if len(model.inputs) != 1:
             raise RuntimeError('Wav2Vec must have one input')
         self.input_tensor_name = model.inputs[0].get_any_name()
-        model_input_shape = model.inputs[0].shape
+        self.dynamic = model.is_dynamic() and dynamic_flag and device == 'CPU'
```

Comment:
The check `device == 'CPU'` doesn't work for HETERO case, better to use check `device in set(parse_devices(device))` (if it is actually should be dynamic for HETERO case)

Reply:
I think we should not take care about which device used, if user says that he want run dynamic model and device does not support it, it get error from runtime, in this case, in future when other devices will support dynamic shapes you do not need modify demo for enabling for them

Reply:
BTW possibly we can add some info about dynamic models support into supported_devices